src/main/java/nextstep/fp/Conditional.java

```diff
@@ -0,0 +1,5 @@
+package nextstep.fp;
+
+public interface Conditional {
```

Comment:
함수형 인터페이스 인지 명시적으로 표시해주세요 :) 
`@FunctionalInterface` 어노테이션을 사용하면 인터페이스에 메소드가 추가됐을때 오류가 발생하기 때문에 미리 오류를 캐치 할 수 있어요. 
```suggestion
@FunctionalInterface
public interface Conditional {
```